widgets/src/google/Common.js

```diff
@@ -90,8 +111,28 @@
                     italic: this._legendFontItalic
                 }
             },
+            series: initSeries(this.getNumSeries()),
+            axes: {
```

Comment:
Yes - but Pie Charts don't - https://developers.google.com/chart/interactive/docs/gallery/piechart and your adding axes to the base class that common to _all_ google charts.